src/components/ActivityFeed/TabNavigation.tsx

```
@@ -0,0 +1,383 @@
+					.join('-')
+					.toLowerCase(),
+				label: trackName.split(/(?=[A-Z])/).join(' '),
+				posts: postsCount
+			});
+		});
+	}
+	const dynamicGroups = ['Governance', 'Treasury', 'Whitelist'];
+	const filteredGroups = tabItems.reduce(
+		(acc: { [key: string]: string[] }, item) => {
+			if (dynamicGroups.includes(item.group)) {
+				(acc[item.group] = acc[item.group] || []).push(item.key);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)